src/assets/sass/demos/card.scss

```diff
@@ -8,3 +8,7 @@
 @import '../protocol/includes/lib';
 @import '../docs/protocol';
 @import '../docs/protocol-extra';
+
+.mzp-t-dark {
```

Comment:
Demos should probably use the "protosite" prefix or some other sort of prefix. Especially with classes like this one which will probably end up in the system at some point.

Reply:
Hmm, I'm not sure here - the demo is intended as an instructional example of how to use the component, so I don't think it's something related to the docs site itself.